README.md

```diff
@@ -14,83 +16,171 @@ yarn add --dev eslint-config-yscope
+- `JestConfig`: A specialized ESLint configuration for Jest files.
+- `MeteorConfig`: A specialized ESLint configuration for Meteor projects.
+
+The other configurations are provided as a group of configuration objects stored in an array. To use
+these configurations, you need to utilize the "spread" (`...`) syntax to unpack the configuration
+objects into your own configuration array:
+- `StylisticConfigArray`: A configuration for stylistic formatting. It is typically combined with
+  `CommonConfig`, as shown in the [ESLint config examples](#eslint-config-examples).
+- `ReactConfigArray`: A specialized ESLint configuration for React applications.
+- `TsConfigArray`: A specialized ESLint configuration for TypeScript projects. A helper
+  `createTsConfigOverride` is also provided. See [ESLint config examples](#eslint-config-examples) for
+  details on how to set it up.
+
+
+### ESLint config examples
+
+1. For use with a CommonJS Node.js project:
+    ```javascript
+    import globals from "globals";
+    
+    import CommonConfig from "eslint-config-yscope/CommonConfig.mjs";
+    import StylisticConfigArray from "eslint-config-yscope/StylisticConfigArray.mjs";
+    
+    
+    const EslintConfig = [
+        CommonConfig,
+        ...StylisticConfigArray,
+        {
+            languageOptions: {
+                globals: {
+                    ...globals.node,
+                },
+            },
+        },
+        {
+            rules: {
+                // Your own overrides
+            },
+        },
+    ];
+    
+    
+    export default EslintConfig;
+    ```
+
+2. For use with a JavaScript React.js project:
+    ```javascript
+    import CommonConfig from "eslint-config-yscope/CommonConfig.mjs";
+    import ReactConfigArray from "eslint-config-yscope/ReactConfigArray.mjs";
+    import StylisticConfigArray from "eslint-config-yscope/StylisticConfigArray.mjs";
+    
+    
+    const EslintConfig = [
+        CommonConfig,
+        ...StylisticConfigArray,
+        ...ReactConfigArray,
+    ];
+    
+    
+    export default EslintConfig;
+    ```
+
+3. For use with a TypeScript React.js project:
+    ```javascript
+    import CommonConfig from "eslint-config-yscope/CommonConfig.mjs";
+    import ReactConfigArray from "eslint-config-yscope/ReactConfigArray.mjs";
+    import StylisticConfigArray from "eslint-config-yscope/StylisticConfigArray.mjs";
+    import TsConfigArray from "eslint-config-yscope/TsConfigArray.mjs";
+    
+    
+    const EslintConfig = [
+        CommonConfig,
+    
+        // NOTE: The map below must be placed directly before the spread `StylisticConfigArray`, to
+        // disable conflicting TsEsLint rules by the Stylistic plugin's "disable-legacy" preset.
+        ...TsConfigArray.map(
+            (config) => ({
+                files: [
+                    "**/*.ts",
+                    "**/*.tsx",
+                ],
+                ...config,
+            })
+        ),
+        ...StylisticConfigArray,
```

Comment:
```suggestion
        ...TsConfigArray.map(
            (config) => ({
                files: [
                    "**/*.ts",
                    "**/*.tsx",
                ],
                ...config,
            })
        ),

        // NOTE: `StylisticConfigArray` must be placed after `TsConfigArray` to override stylistic
        // rules within `TsConfigArray` (we can't remove the stylistic rules from `TsConfigArray`
        // since they come from a preset that `TsConfigArray` extends).
        ...StylisticConfigArray,
```